mod/account/assets/script/toggle_visibility.js.coffee

```diff
@@ -0,0 +1,12 @@
+$ ->
+  $("body").on "click", "._toggle-pw-visibility", (event) ->
+    passwordField = $("._pw-input")
+    togglePassword = $("._pw-text-area")
+
+    if passwordField.prop("type") is "password"
+      passwordField.prop("type", "text")
```

Comment:
not a big deal, but it's slightly more idiomatic not to use parentheses here. We're not using any style rules on coffeescript for now. Maybe someday soon?